lib/api/BackbeatAPI.js

```diff
@@ -482,18 +571,37 @@ class BackbeatAPI {
         });
     }
 
+    _findAndDeleteMember(member, site, cb) {
+        const keys = this._getSortedSetKeys(site);
+        let i = 0;
+        let shouldContinue = true;
+        return async.doWhilst(next => {
+            const key = keys[i++];
+            this._redisClient.zscore(key, member, (err, res) => {
```

Comment:
👍 I was just thinking using `zrem` on each set, but this should be a faster overall, nice!

Reply:
Nice to consider the best-case scenario, too! 💃